app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,128 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.exception.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.repository.TaskRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class DemoHttpHandler implements HttpHandler {
+    private TaskRepository taskRepository = new TaskRepository();
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private final String TASK_DEFAULT_PATH = "/tasks";
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        URI uri = exchange.getRequestURI();
+        String path = uri.getPath();
+        String body = getResponseBody(exchange);
+
+        System.out.println(method + " " + path);
+
+        try {
+            // GET /tasks
+            if (method.equals("GET") && path.equals(TASK_DEFAULT_PATH)) {
+                List<Task> tasks = taskRepository.findAll();
+                sendResponse(exchange, toJson(tasks), HttpStatus.OK);
+                return;
+            }
+
+            // POST /tasks
+            if (method.equals("POST") && path.equals(TASK_DEFAULT_PATH)) {
+                Task task = toTask(body);
+                Task savedTask = taskRepository.save(task);
+                sendResponse(exchange, toJson(savedTask), HttpStatus.CREATED);
+                return;
+            }
+
+            //GET /tasks/{id}
+            if (method.equals("GET") && containPathVariable(path)) {
+                Long id = getPathVariable(path);
+                Task task = findTaskById(id);
+                sendResponse(exchange, toJson(task), HttpStatus.OK);
+                return;
+            }
+
+            // PUT /tasks/{id}
+            if (method.equals("PUT") && containPathVariable(path)) {
+                Long id = getPathVariable(path);
+                Task task = findTaskById(id);
+                Task taskToUpdate = toTask(body);
+                task.setTitle(taskToUpdate.getTitle());
```

Comment:
```suggestion
                Task target = findTaskById(id);
                Task source = toTask(body);
                target.setTitle(source.getTitle());
```

target <-> source 을 사용해서 표현해보았는데 어떤 것 같으세요?

Reply:
정말 좋은 변수명 같아요! 이제부터 자주 사용할 변수명이 될거 같네요! 감사합니다 :)